src/components/RegisterLoginModal/RegisterLoginModal.jsx

```diff
@@ -1,15 +1,34 @@
-import { useState } from 'react';
-
+/* eslint-disable react/no-children-prop */
+/* eslint-disable no-shadow */
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-absolute-path */
+/* eslint-disable import/no-unresolved */
+import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import Modal from '@mui/material/Modal';
-import Box from '@mui/material/Box';
-import RegisterLoginHeader from '@/components/RegisterLoginHeader/RegisterLoginHeader';
+import { styled } from '@mui/material/styles';
+import { useState } from 'react';
+
+import mapError from '/src/utils/services/mapError';
 import LoginForm from '@/components/LoginForm/LoginForm';
 import RegisterForm from '@/components/RegisterForm/RegisterForm';
-import ActionAlert from '../ActionAlert/ActionAlert';
+import RegisterLoginHeader from '@/components/RegisterLoginHeader/RegisterLoginHeader';
 
+import theme from '../../theme/theme';
```

Comment:
As it's in the context, it should be imported like this: https://mui.com/styles/advanced/#accessing-the-theme-in-a-component